llvm/test/tools/llvm-objcopy/COFF/subsystem.test

```diff
@@ -30,6 +30,21 @@
 # INVALID-MAJOR-NUMBER: 'foo' is not a valid subsystem major version
 # INVALID-MINOR-NUMBER: 'bar' is not a valid subsystem minor version
 
+# RUN: llvm-objcopy --subsystem=efi_application %t.in.exe - | llvm-readobj --file-headers - | FileCheck %s --check-prefix=EFI-APPLICATION
```

Comment:
The pattern elsewhere in this test file, and I believe more generally in llvm-objcopy tests, is to have the llvm-readobj command in a separate invocation, with a file on disk created. This makes it easier to inspect the llvm-objcopy output should the need arise when debugging a test failure.
```suggestion
# RUN: llvm-objcopy --subsystem=efi_application %t.in.exe %t.out.exe
# RUN: llvm-readobj --file-headers %t.out.exe | FileCheck %s --check-prefix=EFI-APPLICATION
```

Same applies for each other case below.